lib/themes/dosomething/paraneue_dosomething/scss/patterns/_cta.scss

```diff
@@ -50,48 +50,103 @@
       @include clearfix;
       display: table;
       padding: $base-spacing / 2 0;
+    }
 
-      @include media($medium) {
-        @include span(12);
-        padding: $base-spacing / 2 0;
+    // Layout for campaigns with a scholarship.
+    &.with-scholarship {
+      .cta__action {
+        width: auto;
+
+        @include media($larger) {
+          @include span(6 of 12 no-gutters);
+          float: none;
+        }
+
+        .cta__button {
+          @include span(3 of 6 no-gutters);
+          display: table-cell;
+          float: none;
+          vertical-align: middle;
+
+          @include media($larger) {
+            width: auto;
+          }
+        }
       }
 
-      @include media($larger) {
-        @include span(6);
-        padding: $base-spacing / 2 0;
+      .cta__message {
+        display: none;
+
+        @include media($larger) {
+          display: table-cell;
+          @include span(6 of 12 no-gutters);
+          float: none;
+        }
       }
     }
 
-    .cta__button {
-      @include span(6 of 12);
-      display: table-cell;
-      float: none;
-      padding: 0;
-
+    // Layout for campaigns without a scholarship.
+    .cta__action {
       @include media($medium) {
-        @include span(4 of 12);
+        display: table-cell;
         text-align: left;
+        width: 145px;
```

Comment:
@DFurnes Again, this was to avoid the button text wrapping as the screen size got smaller. If I gave it a width percentage, like 25%, as the screen got smaller 25% of the nav would get too small and the button text would wrap.